govcd/external_network_v2.go

```diff
@@ -215,6 +215,9 @@ func (vcdClient *VCDClient) GetTier0RouterInterfaceByName(externalNetworkId, dis
 	if err != nil {
 		return nil, fmt.Errorf("error getting %s by DisplayName '%s':%s", labelNsxtTier0RouterInterface, displayName, err)
 	}
+	if len(allT0Interfaces) == 0 {
```

Comment:
Making T0 Interface lookups throw `ErrorEntityNotFound` to conform with standard (and also fix broken datasource test)